# [\#9825 PR](https://github.com/ethereum/solidity/pull/9825) `merged`: CircleCI Windows test run.

#### <img src="https://avatars.githubusercontent.com/u/1347491?v=4" width="50">[ekpyron](https://github.com/ekpyron) opened issue at [2020-09-16 14:25](https://github.com/ethereum/solidity/pull/9825):



#### <img src="https://avatars.githubusercontent.com/u/1347491?v=4" width="50">[ekpyron](https://github.com/ekpyron) commented at [2020-09-16 15:00](https://github.com/ethereum/solidity/pull/9825#issuecomment-693464299):

Could it really be that my blindly written first attempt actually worked :-)?

#### <img src="https://avatars.githubusercontent.com/u/1347491?v=4" width="50">[ekpyron](https://github.com/ekpyron) commented at [2020-09-16 15:01](https://github.com/ethereum/solidity/pull/9825#issuecomment-693465157):

Ah, I at least should have soltest output the test result artifacts for uploading.

#### <img src="https://avatars.githubusercontent.com/u/1347491?v=4" width="50">[ekpyron](https://github.com/ekpyron) commented at [2020-09-16 16:23](https://github.com/ethereum/solidity/pull/9825#issuecomment-693516550):

Hm... that's not a good sign... right now the ``b_win`` run has already been completed for 5 minutes, but the ``t_win`` run is still marked as "blocked" and hasn't started, not exactly sure why (it did start before the last change, which was only a rebase)...

#### <img src="https://avatars.githubusercontent.com/u/1347491?v=4" width="50">[ekpyron](https://github.com/ekpyron) commented at [2020-09-16 16:24](https://github.com/ethereum/solidity/pull/9825#issuecomment-693517011):

Ah, now it started... but only around 7 minutes after ``b_win`` was done... I hope this won't become a problem and won't end up in AppVeyor-like queuing issues later on...

#### <img src="https://avatars.githubusercontent.com/u/1347491?v=4" width="50">[ekpyron](https://github.com/ekpyron) commented at [2020-09-16 16:27](https://github.com/ethereum/solidity/pull/9825#issuecomment-693518791):

What also comes to my mind just now: I haven't checked what actually happens, if a command in those powershell script fails :-). I'll try that here later on by adding a failing test or something like that - not sure powershell has or needs something like ``set -e`` to properly report CI errors.

#### <img src="https://avatars.githubusercontent.com/u/1347491?v=4" width="50">[ekpyron](https://github.com/ekpyron) commented at [2020-09-17 11:05](https://github.com/ethereum/solidity/pull/9825#issuecomment-694159768):

Ok, apparently failures are in fact *not* propagated correctly as CI status. Will try to figure out how to fix that.

#### <img src="https://avatars.githubusercontent.com/u/1347491?v=4" width="50">[ekpyron](https://github.com/ekpyron) commented at [2020-09-17 13:31](https://github.com/ethereum/solidity/pull/9825#issuecomment-694236212):

For the record: I checked earlier that the CI runs properly fail now with an intentional compiler error and an intentional test error and it seemed to work now.


-------------------------------------------------------------------------------



[Export of Github issue for [ethereum/solidity](https://github.com/ethereum/solidity). Generated on 2024.12.15 at 06:45:24.]
